src/app/carbon-estimation-treemap/carbon-estimation-treemap.component.html

```diff
@@ -2,26 +2,28 @@
   <span class="tce-chart-total-colour"></span>
   <span class="apexcharts-legend-text" id="tce-chart-total-text">Total Emissions Estimate {{ totalEmissions() }}</span>
   <!--span><button (click)="toggleMassPercentages()">toggle</button></span-->
-  <span class="tce-switch-container" *ngIf="shouldShowUnitsSwitch()">
-    <label class="tce-switch-label" [class.tce-switch-active]="isMass()">
-      <input
-        type="radio"
-        name="massPercentSwitch"
-        [checked]="isMass()"
-        (change)="isMass.set(true)"
-        class="tce-switch-input" />
-      kg
-    </label>
-    <label class="tce-switch-label" [class.tce-switch-active]="!isMass()">
-      <input
-        type="radio"
-        name="massPercentSwitch"
-        [checked]="!isMass()"
-        (change)="isMass.set(false)"
-        class="tce-switch-input" />
-      %
-    </label>
-  </span>
+  @if(shouldShowUnitsSwitch()) {
```

Comment:
I presume this and the other html changes in files are fixing things, rather than strictly about isolating the styles